packages/server/src/fhir/operations/projectclone.ts

```diff
@@ -72,6 +77,38 @@ class ProjectCloner {
     return newProject as Project;
   }
 
+  isResourceAllowed(resource: Resource): boolean {
+    if (resource.resourceType === 'Project') {
+      return true;
+    }
+    if (!this.isAllowedResourceType(resource.resourceType)) {
+      return false;
+    }
+
+    if (!this.isAllowedResourceId(resource.id as string)) {
+      return false;
+    }
+
+    return true;
+  }
+
+  isAllowedResourceId(resourceId: string): boolean {
+    if (this.includeIds.length > 0 && !this.includeIds.includes(resourceId)) {
+      return false;
+    }
+    if (this.excludeIds.length > 0 && this.excludeIds.includes(resourceId)) {
```

Comment:
Minor:  I appreciate the aesthetic symmetry of the `length > 0` check, but it's not necessary in this case (I think?)

Reply:
Yes! It's not necessary for excludeIds.